src/OrchardCore.Modules/OrchardCore.ContentFields/Indexing/ContentPickerFieldIndexHandler.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.Indexing;
+
+namespace OrchardCore.ContentFields.Indexing
+{
+    public class ContentPickerFieldIndexHandler : ContentFieldIndexHandler<ContentPickerField>
+    {
+        public override Task BuildIndexAsync(ContentPickerField field, BuildFieldIndexContext context)
+        {
+            var options = context.Settings.ToOptions();
```

Comment:
I don't think we can configure it in this case. I would just force the options to remove `Analyze`, `Sanitize` and to add `Store`. Look for examples in other handlers. Because in this case we really care about the technical value and we don't want to alter it, or not store it.

Reply:
fixed